sidebars.js

```diff
@@ -11,21 +11,37 @@
 
 // @ts-check
 
-/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
-const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{ type: 'autogenerated', dirName: '.' }],
-
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
-    {
-      type: 'category',
-      label: 'Tutorial',
-      items: ['hello'],
-    },
-  ],
-   */
+const swap = (items, itemsWithHref) => {
+  const result = items.map((item) => {
+    if (item.type === 'category') {
+      return { ...item, items: swap(item.items, itemsWithHref) }
+    }
+
+    if (itemsWithHref[item.id]) {
+      item = {
+        type: 'link',
+        label: itemsWithHref[item.id].title,
```

Comment:
Why not just `item.label`?

Reply:
Because `item` only has `id` and `type` here, as it's already been through the default sidebar generator.

I did ponder if I could edit the items _before_ running through the default generator but I wasn't sure if the default generator would support `link` types. I'll have a play, see if it makes the code cleaner (and if it works).

Reply:
I need to run the default generator first, because it has no concept of `link` items in it, meaning it'd ignore any fiddling I did with the original arguments.

https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-plugin-content-docs/src/sidebars/generator.ts